app/renderer/components/task_suite.js

```diff
@@ -5,6 +5,7 @@ const EventEmitter = require('events');
 const TaskCard = require('./task_card');
 const AddTask = require('./add_task');
 const AppStatus = require('../app_status');
+const Draggable = require('vuedraggable');
```

Comment:
Please, move the external dependency (vuedraggable) to the top, just for ordering :)